### ISSUE TITLE
Paused player prevents Windows from going to "sleep".

### ISSUE THREAD

**Issue by Zuko** [jellyfin/jellyfin-media-player#141]
**Describe the bug**
Paused player prevents Windows from going to "sleep".

**To Reproduce**
1. Set "Sleep after time" to 1min in Power Settings
2. Pause movie
3. Wait
4. Watch Windows not going to "sleep"

**Expected behavior**
Windows going to sleep after configured time when movie is paused.

**Screenshots**
None

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: 10
 - Installation Method: Windows installer

**Additional context**
```
C:\WINDOWS\system32>powercfg -requests
DISPLAY:
[PROCESS] \Device\HarddiskVolume4\Program Files\Jellyfin\Jellyfin Media Player\JellyfinMediaPlayer.exe

SYSTEM:
[PROCESS] \Device\HarddiskVolume4\Program Files\Jellyfin\Jellyfin Media Player\JellyfinMediaPlayer.exe

AWAYMODE:
None.

EXECUTION:
None.

PERFBOOST:
None.

ACTIVELOCKSCREEN:
None.
```

**Comment by iwalton3** [comment]
Right now I believe this is by design in that it only prevents sleeping when something is playing.

**Comment by Zuko** [comment]
> only prevents sleeping when something is playing.

No, it prevents sleeping when player is "active" (paused or not). When I go to "Library" Windows is going to sleep after configured time.

**Comment by sevenrats** [comment]
looks like this was been resolved for some time by #276 
feel free to comment on this issue or open a new one if the issue persists.
